siliconcompiler/remote/client.py

```diff
@@ -355,6 +354,27 @@ def check_progress(chip):
         return [], True
 
 
+###################################
+def _update_entry_manifests(chip):
+    '''
+    Helper method to update locally-run manifests to include remote job ID.
+    '''
+
+    flow = chip.get('option', 'flow')
+    jobid = chip.get('record', 'remoteid')
```

Comment:
Having `jobid`, `remoteid` and `job_hash` gets a bit confusing to me at this point.

Reply:
at some point we should go through and ensure the naming is consistent (remoteid) is the correct term now

Reply:
SC-615